analytics/src/test/java/com/segment/analytics/EdgeFunctionMiddlewareTest.kt

```diff
@@ -108,7 +108,7 @@ class EdgeFunctionMiddlewareTest {
         analytics.track("foo")
 
         val privateEdgeFunctions: Field = Analytics::class.java.getDeclaredField("edgeFunctionMiddleware")
-        Assertions.assertThat(privateEdgeFunctions).isNotNull
+        Assertions.assertThat(privateEdgeFunctions).isNotNull()
```

Comment:
good catch :)

Reply:
Yay! I wanted to look at the new test and then asked Cody what to do when the error was there.

Reply:
surprising how the CI was passing without this 🤔

Reply:
Yah they pass locally too without the parens, that's why I didn't catch it, but gives some weird non-compile warning.